src/service-module/make-base-model.ts

```diff
@@ -176,6 +176,19 @@ export default function makeBaseModel(options: FeathersVuexOptions) {
       return this._getters('find', params)
     }
 
+    public static count() {
+      const params = {
+        $limit: 0 // <- limit 0 in feathers is a fast count query
```

Comment:
Tiny correction: `$limit` needs to be wrapped in the `query`
```suggestion
        query: { $limit: 0 } // <- limit 0 in feathers is a fast count query
```

Reply:
Good catch! Thanks